src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/Services/ElasticSearchService.cs

```diff
@@ -53,13 +57,20 @@ ILogger<ElasticsearchService> logger
 
     public async Task<SearchResult> SearchAsync(string indexName, string term, int start, int pageSize)
     {
+        var result = new SearchResult();
+
+        if (!_elasticConnectionOptions.IsFileConfigurationExists())
+        {
+            _logger.LogWarning("Elasticsearch: Couldn't execute search. The Elasticsearch has yet been configured.");
```

Comment:
Consistency

Elasticsearch has not been configured yet

Reply:
too late

Reply:
I'll update it.